src/Hl7.Fhir.Core/Rest/FhirClient.cs

```diff
@@ -182,25 +173,12 @@ public ParserSettings ParserSettings
             get => Settings.ParserSettings;
             set => Settings.ParserSettings = value;
         }
-
-        /// <summary>
-        /// Returns the HttpWebRequest as it was last constructed to execute a call on the FhirClient
-        /// </summary>
-        public HttpWebRequest LastRequest { get { return LastClientRequest as HttpWebRequest; } }
-
-        /// <summary>
-        /// Returns the HttpWebResponse as it was last received during a call on the FhirClient
-        /// </summary>
-        /// <remarks>Note that the FhirClient will have read the body data from the HttpWebResponse, so this is
-        /// no longer available. Use LastBody, LastBodyAsText and LastBodyAsResource to get access to the received body (if any)</remarks>
-        public HttpWebResponse LastResponse { get { return LastClientResponse as HttpWebResponse; } }
-
         #endregion
 
-        [Obsolete ("OnBeforeRequest is deprecated, please add a HttpClientEventHandler to the constructor to use this functionality", true)]
+        [Obsolete ("OnBeforeRequest is deprecated, please add a HttpClientEventHandler or another HttpMessageHandler to the constructor to use this functionality", true)]
```

Comment:
We should touching these events throw an exception - or remove them completely.

Reply:
The true at the end of Obsolete will cause a compiler warning, so that's good enough.